camelot/admin/admin_route.py

```diff
@@ -41,7 +42,7 @@ def admin_for(cls, route):
         """
         assert isinstance(route, tuple)
         try:
-            admin = cls._admin_routes.resolve(route)
+            admin = naming_context.resolve(route)
         except KeyError:
```

Comment:
is dit nu geen NameNotFoundException ?

Reply:
Yup, fixed in c3fe9e669e3acdd160d853813cb1581dea93c507.